src/kili/adapters/kili_api_gateway/organization/types.py

```diff
@@ -0,0 +1,23 @@
+"""Organization types for Kili API Gateway."""
+
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass(frozen=True)
+class KiliAPIGateWayCreateOrganizationInput:
+    """Organization to create Kili API Gateway input."""
+
+    name: str
+    address: str
+    city: str
+    country: str
+    zip_code: str
```

Comment:
since they are all str, we should make this dataclass kwargs_only, so that we  avoid interchanging the fields

Reply:
👍 when it will be part of the minimal version we support